drupal-ukraine/dmaps
# VM D7 and D8

- added cibox vm
- drupal 7
- drupal 8

Steps for review: 
- download branch 
- vagrant up
- check vm is working(`vagrant up` and then `vagrant ssh`)
- check D7 (`cd /var/www/d7` and run `sh reinstall.sh`) should be installed
- check D8 ( http://drupal8.192.168.56.132.xip.io ) can be installed
- merge
